sqlagg/base.py

```diff
@@ -175,6 +179,12 @@ def table_filter(table_name, metadata):
 
         return self._metadata
 
+    def count(self, connection, filter_values=None):
```

Comment:
if this is going to be used with https://github.com/dimagi/commcare-hq/pull/10614/ to replace https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/reports/util.py#L17 then doesn't it need to sum?